crates/router/src/connector/utils.rs

```diff
@@ -211,6 +221,14 @@ impl<Flow, Request, Response> RouterData for types::RouterData<Flow, Request, Re
             .and_then(|a| a.address.as_ref())
             .ok_or_else(missing_field_err("shipping.address"))
     }
+
+    fn get_shipping_address_with_phone_number(&self) -> Result<&api::Address, Error> {
```

Comment:
get_shipping_address is already present, do we still need this new method?

Reply:
We need this because get_shipping_address() will return `AddressDetails` struct, we require `Address` struct